src/Symfony/Component/Mime/Tests/Test/Constraint/EmailAttachmentCountTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Tests\Test\Constraint;
```

Comment:
This namespace looks wrong to me
```suggestion
namespace Symfony\Component\Mime\Tests\Constraint;
```

cc @xabbuh @derrabus

Reply:
I agree, I wasn't sure and in doubt, preferred to match the actual dir structure. Will update

Reply:
Hmm, actually this is how it's done elsewhere, in DomCrawler for example :thinking:

Reply:
* We use Symfony\Component\Mime\Test for Abstract test cases etc, which are covered by BC
* We use Symfony\Component\Mime\TestS for the tests itself

I never saw Symfony\Component\Mime\Tests\Test 🤔

Reply:
Ok, the constraint is in Test namespace

Reply:
These are the tests of the custom test constraints, indeed covered by BC. Another example of this can be found here: https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/DomCrawler/Tests/Test/Constraint/CrawlerAnySelectorTextContainsTest.php. The `Tests\Test\Constraint` bit is used here